components/PyTorch/pytorch-kfp-components/templates/ax_train_component.yaml

```diff
@@ -17,24 +17,21 @@ description: |
 
 inputs:
   - {name: input_data, description: 'Input dataset path'}
-  - {name: cifar_script_args, description: 'Arguments to the cifar script'}
+  - {name: script_args, description: 'Arguments to the cifar script'}
   - {name: ptl_arguments, description: 'Arguments to pytorch lightning Trainer'}
-  # Uncomment below inputs for ax hpo
-  # - {name: trial_id, type: Integer, description: 'Trial Id'}
-  # - {name: model_parameters, type: JsonObject, description: 'Model parameters for trainer'}
-  # - {name: results, type: String, description: 'Path to write training results'}
+  - {name: trial_id, type: Integer, description: 'Trial Id'}
+  - {name: model_parameters, type: JsonObject, description: 'Model parameters for trainer'}
+  - {name: results, type: String, description: 'Path to write training results'}
 outputs:
   - {name: tensorboard_root, description: 'Tensorboard output path'}
   - {name: checkpoint_dir, description: 'Model checkpoint output'}
   - {name: MLPipeline UI Metadata, description: 'MLPipeline UI Metadata output'}
   - {name: MLPipeline Metrics, description: 'MLPipeline Metrics output'}
-  
-  
+
+
 implementation:
   container:
-    # For GPU use
-    # image: public.ecr.aws/pytorch-samples/kfp_samples:latest-gpu
-    image: public.ecr.aws/pytorch-samples/kfp_samples:latest
+    image: public.ecr.aws/pytorch-samples/kfp_samples:latest-gpu
     command: ["python3", "cifar10/cifar10_pytorch.py"]
```

Comment:
In the templates, this can be mentioned as a comment instead of pointing to actual example

Reply:
Updated the string